src/basic/selinux-util.c

```diff
@@ -244,9 +244,14 @@ int mac_selinux_apply(const char *path, const char *label) {
         assert(label);
 
         if (setfilecon(path, label) < 0) {
-                log_enforcing_errno(errno, "Failed to set SELinux security context %s on path %s: %m", label, path);
+                const int saved_errno = errno;
```

Comment:
`const` appears unnecessary in inner blocks like this...

Reply:
dropped `const` specifier